.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Generate and Release CRD Docs
+
+on:
+  push:
+    tags:
+      - '*'  # Trigger on tag pushes
+
+jobs:
+  generate-release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write  # Needed for creating releases and uploading assets
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.25'
```

Comment:
Using a version from `go.mod` should be more future-proof. `go-version-file: 'path/to/go.mod'`

Reply:
applied in https://github.com/kyma-project/kyma-infrastructure-manager/pull/1084/commits/a9497c2b075eec9b1f4ebc46362bf2ca533b3c2c